src/services/quota/index.tsx

```diff
@@ -1,87 +1,114 @@
 import { IS_MOCK } from "../../config";
 
-export interface Transaction {
+export interface QuotaItem {
   category: string;
   quantity: number;
-  transactionTime: number;
+  transactionTime?: number;
 }
 
 export interface Quota {
-  remainingQuota: Transaction[];
+  remainingQuota: QuotaItem[];
 }
 
 export interface PostTransaction {
```

Comment:
Should we rename this as GroupTransaction instead? Might be clearer haha

Reply:
As discussed, we'll use the current approach since it may not be a group transaction. I've also added back `Transaction`